### DISCUSSION TITLE
Using radarr/sonarr with rdt-clinet’s internal download client

### DISCUSSION THREAD

**Discussion by FutureTense** [rogerfar/rdt-client#500] [General]
I’m using both sonarr and radarr with rdt-client’s internal qbittorrent.  Both are configured identically (except for category name) and sonarr seems to be having no problem.  However radarr is telling me

> You are using docker; download client qBittorrent places downloads in /data/downloads/radarr but this directory does not appear to exist inside the container. Review your remote path mappings and container volume settings.

How can I log into this qbittorrent instance?  What/where is the username/password?

**Comment by rogerfar** [comment]
You have to use the "mapping" feature here.

You download it to a certain path, but you have to trick radarr/sonarr that it's somewhere else. You do this by setting that setting.

**Reply by franchiceddhu** [reply]
Hi there, I'm sorry to bother you but I still don't understand. What should I write inside here? I'm using rdt-client on Docker on my Unraid. What folder should I put for mapped path?

**Reply by get-loose** [reply]
Rdt-client downloads files inside it's dockercontainer to a "inside docker"-folder. That folder is mapped (see rdt-client setup) in unraid to a "real outside docker"-folder where the downloads reside. The same needs to happen for radarr : inside the radarr container (see your error message above) there needs to be a folder /data/downloads/radarr where radarr finds the downloads.  Map this folder (/data/downloads/radarr) to the real outside-docker folder where the files are downloaded.

Edit :  Assuming both radarr and rdt-client run in a container, if you set your 2x docker setup mappings correctly, then you don't need to use the mapping in radarr itself. (Look at your sonarr because that is working.).
